packages/frontend/src/pages/NameForm.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { FullWidthText } from "../components/Input/Text";
+import TextButtonInput from "../components/Input/TextButtonInput";
+
+export default function NameForm() {
+	const navigate = useNavigate();
+
+	return (
+		<div className="flex flex-col space-y-normal justify-center w-5/6">
+			<FullWidthText>What's your name?</FullWidthText>
+			<TextButtonInput
+				placeholder="Chickie McDee"
+				buttonText="let's go -->"
+				onClick={(input) => navigate(`/flock/${input}/lobby/`)}
```

Comment:
This onclick should be using the coop_name, not the input chick's name - maybe just hard code with a placeholder?